README.md

```diff
@@ -171,43 +173,35 @@ You can also found different example on how to create a Key Vault account:
 ### Ask Key Vault to create a certificate for you
 
 ```javascript
-    keyVaultClient.createCertificate(vaultObj.properties.vaultUri, certificateName, { certificatePolicy: certificatePolicy });
+    await certificateClient.beginCreateCertificate(certificateName, certificatePolicy);
```

Comment:
beginCreateCertificateAndWait ?

Reply:
Don't have such method, I use the method `pollUntilDone()` to resolve it.